hermeto/core/models/input.py

```
@@ -209,8 +209,37 @@ class PipBinaryFilters(BinaryModeOptions):
+    platform: Optional[str] = None
+
+    @pydantic.field_validator("py_version")
+    @classmethod
+    def _validate_python_version(cls, py_version: Optional[str]) -> Optional[int]:
+        if py_version is None:
+            return None
+
+        try:
+            version = int(py_version.replace(".", ""))
+        except ValueError:
+            raise ValueError(f"Invalid Python version: {py_version}")
```

Ruff (0.14.0)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)